datasetinsights/datasets/synthetic.py

```diff
@@ -16,10 +16,13 @@
     Captures,
 )
 from datasetinsights.datasets.unity_perception.tables import SCHEMA_VERSION
+from datasetinsights.datasets.unity_perception.usim import (
```

Comment:
Why has this usim.py be moved to datasetinsights.datasets.unity_perception? @sanjayuconn